vars/postSuccessfulMetrics.groovy

```diff
@@ -9,16 +9,30 @@ Post multiple metrics
        This will use current time - current build start time
   e.g. startTimeInMillis = xxxxx
        This will use current time - xxxx
+- appendBranchSuffix defines whether we want to additionally add whether it is
+  a master or pr build
+  e.g. appendBranchSuffix = true
+       selfservice.master or selfservice.PR
+  e.g. appendBranchSuffix = false
+       metricPrefix remains unchanged
 */
-def call(String metricPrefix, long startTimeInMillis = -1) {
+def call(String metricPrefix, long startTimeInMillis = -1, boolean appendBranchSuffix = false) {
```

Comment:
this feels a bit tangled up to me. I think we just want small functions that do one thing  well. So we could have a separate function called 'appendBranchSuffix' that does this logic, but we should then call that from Jenkinsfile, so it would be 
```
successfulMetrics(appendBranchSuffix('connector'))
```
or whatever